README.md

```diff
@@ -99,9 +149,13 @@ open-attestation wrap ./examples/raw-documents/ ./examples/wrapped-documents/ --
 open-attestation wrap ./examples/raw-documents/ ./examples/wrapped-documents/ --oav3
 ```
 
-## Document privacy filter
+:::note
+For transferable records, you should wrap them individually as each of them would be minted to a unique title escrow that represents the beneficiary and holder entities of the document. For more information about title escrow, refer [here](/docs/integrator-section/transferable-record/title-escrow).
+:::
```

Comment:
:::note doesn't seem to render the markdown properly

Reply:
ya does not work in github readme. I have changed it to another one like a block of text. This way also works in the documentation-website.